docs/conf.py

```diff
@@ -172,4 +176,19 @@
 ]
 
 
+github_doc_root = 'https://github.com/apiaryio/api-elements/tree/master/doc/'
+github_doc_root = 'https://github.com/apiaryio/api-elements/tree/master/doc/'
+
+
+def setup_recommonmark(app):
+    app.add_config_value('recommonmark_config', {
+        'url_resolver': lambda url: github_doc_root + url,
```

Comment:
`github_doc_root + url` could be prone to missing/double slashes - just saying, I believe you tried it out. Otherwise `os.path.join()` might work as an easy drop-in equivalent.